diff --git a/node_modules/@tolki/ts/dist/enums.d.ts b/node_modules/@tolki/ts/dist/enums.d.ts
index b39a983..12a3549 100644
--- a/node_modules/@tolki/ts/dist/enums.d.ts
+++ b/node_modules/@tolki/ts/dist/enums.d.ts
@@ -1,4 +1,4 @@
-import { AsEnum, CaseValue, DefineEnumResult, EnumConst, FromResult } from '../packages/types/src/index.ts';
+import { AsEnum, CaseValue, DefineEnumResult, EnumConst, FromResult } from '@tolki/types';
 export type { AsEnum, CaseValue, DefineEnumResult, EnumConst, FromResult };
 /**
  * Creates an enum instance from a case value similar to PHP's `BackedEnum::from()`
diff --git a/node_modules/@tolki/ts/dist/routes.d.ts b/node_modules/@tolki/ts/dist/routes.d.ts
index 0dd7e15..985d27d 100644
--- a/node_modules/@tolki/ts/dist/routes.d.ts
+++ b/node_modules/@tolki/ts/dist/routes.d.ts
@@ -1,4 +1,4 @@
-import { DefineRouteResult, InferPageProps, InferRequestPayload, RouteArgMeta, RouteCallResult, RouteCallResultWithComponent, RouteComponentType, RouteFormResult, RouteMetadata, RouteQueryOptions } from '../packages/types/src/index.ts';
+import { DefineRouteResult, InferPageProps, InferRequestPayload, RouteArgMeta, RouteCallResult, RouteCallResultWithComponent, RouteComponentType, RouteFormResult, RouteMetadata, RouteQueryOptions } from '@tolki/types';
 export type { DefineRouteResult, InferPageProps, InferRequestPayload, RouteArgMeta, RouteCallResult, RouteCallResultWithComponent, RouteComponentType, RouteFormResult, RouteMetadata, RouteQueryOptions, };
 /**
  * Attach a phantom page props type to a `defineRoute` result without losing
